rttest/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ if(${ament_cmake_FOUND})
 
   add_library(rttest SHARED src/rttest.cpp)
   target_link_libraries(rttest PRIVATE m stdc++ Threads::Threads)
+  target_link_libraries(rttest PUBLIC rt)
```

Comment:
Don't we need to `find_package` something to make this available?  I guess if this literally adds `-lrt` to the gcc command-line, that would make it work, but it doesn't seem very modern CMake-y.

Reply:
So I don't think that it needs to be `find_package`'d because it is a peer to other libraries that come in via `libc` (`rt`, `pthread`, `dl`, `m`, etc).

For amd64, the default conda sysroot is a bit older (based on centos6), which includes `libc 2.12` versus the newer sysroot based on centos7 and uses `libc 2.17`.  The documentation seems to indicate that the newer version doesn't need the explicit linking.

I'm currently in the process of trying this and will report back, it may be that this is completely unnecessary.

Reply:
Yes, I have confirmed with the newer sysroot this is no longer an issue.